index.html

```diff
@@ -0,0 +1,14 @@
+<nav id="navbar">
+    <ul>
+      <li><a href="#welcome-section">welcome section</a><li>
+      <li><a href="#project section">project section</a><li>
+    </ul>
+  </nav>
+  <div id="welcome-section">
+    <h1> Hey I am Mimic </h1>
+  </div>
```

Comment:
- [ ] Nice effort pointing out the use of semantic tags 👍. However, it is good to include an explanation as to why a change is required with reasons why you think they should make the changes. Please, let us edit this comment to be to include why semantics tags should be used over divs.

Reply:
Kindly use HTML5 semantic tags.
This is important because pages made with semantic elements are much easier to read and offers a better user experience..